server/backend/src/users/users.service.ts

```diff
@@ -29,6 +31,42 @@ export class UsersService {
     return user.groups;
   }
 
+  /**
+   * Register a camera to the given user and group
+   * 
+   * @param userId the user's ID.
+   * @param groupId the group ID.
+   * @param name New camera's name
+   * @param pubKey New camera's public key
+   * @param serial New camera's serial
+   */
+  public async putCamera(
```

Comment:
nit: Both the endpoint and the service call should be something like `registerCamera` rather than `putCamera`, "register" makes more sense out of context than "put" at least to me.
```suggestion
  public async registerCamera(
```